### DISCUSSION TITLE
Configurazione multipla di dispositivi Shelly

### DISCUSSION THREAD

**Discussion by aplak1** [indomus/forum#995] [Networking]
Buongiorno,

dispongo di diversi dispositivi Shelly Plug S e Shelly 1.
Ho correttamente configurato tramite Wifi alcuni dei miei dispositivi. Volevo sapere se fosse possibile configurare la rete Wifi Client in modo automatico per una serie di dispositivi, senza dover effettuare la procedura per ogni singolo dispositivo.

Grazie!

**Comment by snakuzzo** [comment]
Puoi provare a vedere se tramite [API Shelly](https://shelly-api-docs.shelly.cloud/gen1/#http-dialect) sia possibile impostare SSID e password...ma comunque dovresti andare a mano a collegarti ad ogni singolo shelly, quindi non penso tu possa farlo in maniera così automatica
